test/liveValidation/swaggers/specification/unknown-rp/unknown.json

```diff
@@ -57,12 +57,7 @@
         "x-ms-pageable": {
           "nextLinkName": "nextLink"
         },
-        "x-ms-odata": "#/definitions/RoleAssignmentFilter",
-        "x-ms-examples": {
-          "GetConfigurations": {
-            "$ref": "./examples/GetAllRoleAssignments.json"
-          }
-        }
+        "x-ms-odata": "#/definitions/RoleAssignmentFilter"
```

Comment:
What's the schema error of this swagger?

Reply:
readOnly property